### PR TITLE
Paolo add home animations

### PR DESCRIPTION
**Pull request by AntonioMateoGomez**
This PR adds animations functionality to the home section.

### CODE REVIEW

File: _sass/components/docs/_navigation.scss [hunk 1]
```diff
@@ -122,7 +122,7 @@
     }
 
     &.navigation-scroll {
-        background: darken($brand-primary, 4%);
+        // background: darken($brand-primary, 4%);
```
Comment (calvellido) [inline _sass/components/docs/_navigation.scss]: Delete code if its purpose is not valid anymore please

File: _sass/components/docs/_navigation.scss [hunk 2]
```diff
@@ -59,7 +59,7 @@
 
           a {
             &:hover {
-              background: darken($brand-primary, 4%);
+              // background: darken($brand-primary, 4%);
```
Comment (calvellido) [inline _sass/components/docs/_navigation.scss]: Delete code if its purpose is not valid anymore please

File: _sass/components/docs/_navigation.scss (hunk 1 above)
Comment (AntonioMateoGomez) [reply _sass/components/docs/_navigation.scss]: Done!

File: _sass/components/docs/_navigation.scss (hunk 2 above)
Comment (AntonioMateoGomez) [reply _sass/components/docs/_navigation.scss]: Done!

File: js/main-hovers.js [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+function commonHoverStyle(id, incubatorHover) {
+  body.style.setProperty('--color-primary', '#F5F7F8');
+  navBrandDark.style.opacity = 0;
+  navBrandWhite.style.opacity = 1;
+  baseArrowLogo.style.opacity = 0;
+  navIconOpenDark.style.opacity = 0;
+  navIconOpenWhite.style.opacity = 1;
+  incubatorBaseLogo.style.opacity = 0;
+  incubatorHover = incubatorHover || false;
+  console.log('incubator Hover');
```
Comment (calvellido) [inline js/main-hovers.js]: This can be removed

File: js/main-hovers.js [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+const categoryRow = document.querySelectorAll('#main-flex div');
+const categoryIconWhite = document.getElementsByClassName('cat-icon-white');
+const categoryIconDark = document.getElementsByClassName('cat-icon-dark');
+const catIconCoreColor = document.getElementById('cat-icon-color-core');
+const catIconFxColor = document.getElementById('cat-icon-color-fx');
+const catIconOpticsColor = document.getElementById('cat-icon-color-optics');
+
+function checkActiveFeature() {
+
+  Array.prototype.map.call(arrowFeatures, function(el) {
```
Comment (calvellido) [inline js/main-hovers.js]: This is a extremely weird way of calling a function :man_shrugging

```suggestion
  Array.map(arrowFeatures, function(el) {
```

File: js/main-hovers.js [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+          break;
+        default:
+          baseHoverStyle();
+      }
+    }
+  });
+}
+
+function setOpacity(iconElements, id, opacity) {
+  Array.prototype.map.call(iconElements, function(obj) {
```
Comment (calvellido) [inline js/main-hovers.js]: This is a extremely weird way of calling a function :man_shrugging

```suggestion
  Array.map(iconElements, function(obj) {
```

File: js/main.js [hunk 6]
```diff
@@ -26,7 +26,125 @@ function scrollFunction() {
...
+  const baseArrowLogo = document.getElementById('base-arrow-animation');
+  baseArrowLogo.addEventListener('load', baseAnimation());
+
+  const incubatorBaseLogo = document.getElementById('incubator-base-animation');
+  incubatorBaseLogo.addEventListener('load', baseAnimation());
+
+  // core elements
+  const corePlayHover = document.getElementById('core');
+
+  corePlayHover.addEventListener('mouseenter', (event) => {
```
Comment (calvellido) [inline js/main.js]: `event` is no tbeing used

File: js/main.js [hunk 7]
```diff
@@ -26,7 +26,125 @@ function scrollFunction() {
...
+    corePlayHover.classList.add('active');
+    checkActiveFeature();
+    arrowCoreAnimation.play();
+    incubatorCoreAnimation.play();
+    arrowBaseAnimation.stop();
+  });
+
+  corePlayHover.addEventListener('mouseleave', (event) => {
```
Comment (calvellido) [inline js/main.js]: `event` is no tbeing used

File: js/main.js [hunk 8]
```diff
@@ -26,7 +26,125 @@ function scrollFunction() {
...
+    baseHoverStyle();
+    arrowCoreAnimation.stop();
+    incubatorCoreAnimation.stop();
+    arrowBaseAnimation.play();
+  });
+
+  // fx elements
+  const fxPlayHover = document.getElementById('fx');
+
+  fxPlayHover.addEventListener('mouseenter', (event) => {
```
Comment (calvellido) [inline js/main.js]: `event` is no tbeing used

File: js/main.js [hunk 9]
```diff
@@ -26,7 +26,125 @@ function scrollFunction() {
...
+    fxPlayHover.classList.add('active');
+    checkActiveFeature();
+    arrowFxAnimation.play();
+    incubatorFxAnimation.play();
+    arrowBaseAnimation.stop();
+  });
+
+  fxPlayHover.addEventListener('mouseleave', (event) => {
```
Comment (calvellido) [inline js/main.js]: `event` is no tbeing used

File: js/main.js [hunk 10]
```diff
@@ -26,7 +26,125 @@ function scrollFunction() {
...
+    baseHoverStyle();
+    arrowFxAnimation.stop();
+    incubatorFxAnimation.stop();
+    arrowBaseAnimation.play();
+  });
+
+  // optics elements
+  const opticsPlayHover = document.getElementById('optics');
+
+  opticsPlayHover.addEventListener('mouseenter', (event) => {
```
Comment (calvellido) [inline js/main.js]: `event` is no tbeing used

File: js/main.js [hunk 11]
```diff
@@ -26,7 +26,125 @@ function scrollFunction() {
...
+    baseHoverStyle();
+    arrowOpticsAnimation.stop();
+    incubatorOpticsAnimation.stop();
+    arrowBaseAnimation.play();
+  });
+
+  // incubator elements
+  const incubatorPlayHover = document.getElementById('incubator');
+
+  incubatorPlayHover.addEventListener('mouseenter', (event) => {
```
Comment (calvellido) [inline js/main.js]: `event` is no tbeing used

File: js/main.js [hunk 12]
```diff
@@ -26,7 +26,125 @@ function scrollFunction() {
...
+
+  opticsPlayHover.addEventListener('mouseenter', (event) => {
+    opticsPlayHover.classList.add('active');
+    checkActiveFeature();
+    arrowOpticsAnimation.play();
+    incubatorOpticsAnimation.play();
+    arrowBaseAnimation.stop();
+  });
+
+  opticsPlayHover.addEventListener('mouseleave', (event) => {
```
Comment (calvellido) [inline js/main.js]: `event` is no tbeing used

File: js/main.js [hunk 13]
```diff
@@ -26,7 +26,125 @@ function scrollFunction() {
...
+  const incubatorPlayHover = document.getElementById('incubator');
+
+  incubatorPlayHover.addEventListener('mouseenter', (event) => {
+    incubatorPlayHover.classList.add('active');
+    checkActiveFeature();
+    incubatorHoverAnimation.play();
+    arrowBaseAnimation.stop();
+  });
+
+  incubatorPlayHover.addEventListener('mouseleave', (event) => {
```
Comment (calvellido) [inline js/main.js]: `event` is no tbeing used

**Pull request merged**